EpiAware/src/EpiAwareUtils/predict.jl

```diff
@@ -0,0 +1,210 @@
+
+"""
```

Comment:
You've got doc strings for a different context?

Reply:
this is the one I pulled inspo from. Once this works can update